nat-lab/tests/utils/network_switcher/network_switcher_windows.py

```diff
@@ -131,3 +134,16 @@ async def _delete_route(self, interface_name: str) -> None:
                 pass
             else:
                 raise exception
+
+    async def _disable_management_interface(self) -> None:
+        if self._interfaces.default is not None:
```

Comment:
is it worth adding an assert so `self._interfaces.default` would be present or do we have cases where that's expected?

Reply:
Yes, if the interface is disabled at the start of the tests, `self._interfaces.default` will be `None`.